test_schemas/en/test_thank_you_census_communal_establishment.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
Shame we don't have unit tests for the handlers, otherwise schemas like these could be avoided.